src/controllers/messagesController.ts

```diff
@@ -159,20 +157,20 @@ export default class MessagesController {
         messageId
       ),
       TE.bindTo("message"),
-      TE.bindW("hasLollipopEnabled", ({ message }) =>
+      TE.bindW("remoteContentConfiguration", ({ message }) =>
         pipe(
-          checkIfLollipopIsEnabled(
-            this.thirdPartyConfigList,
-            user.fiscal_code,
-            message.sender_service_id
+          message.content.third_party_data.configuration_id,
+          TE.fromNullable(
+            ResponseErrorInternal("Cannot get remote content configuration")
```

Comment:
I think we can be more specific here, we cannot retrieve the configuration because of the missing `configuration_id`